app/common/services/maps.js

```diff
@@ -140,4 +163,14 @@ function Maps(ConfigEndpoint, L, _, CONST) {
             popupAnchor: [0, 0 - size[1]]
         });
     }
+
+    // default view is centered on nairobi
+    function defaultValues(config) {
```

Comment:
Here, if you use config.center and config.zoom, you will get the default-values for the deployment, this defaults to Nairobi if the user has not set any own values but sometimes they have (this is set in Settings->General).